mep/accounts/tests/test_accounts_admin.py

```diff
@@ -65,7 +65,13 @@ def test_clean(self):
         assert form.is_valid()
         assert form.cleaned_data['end_date'] == form_data['end_date']
 
-class TestBorrowAdminForm(TestCase):
+
+class TestPartialDateMixin(TestCase):
+
+    # Letting this test stand for the partial date mixin,
+    # since that's the only functionality being checked here,
+    # and it is common to both sets + mixin needs a concrete
+    # case to test against
```

Comment:
This comment as written doesn't make sense to me. Maybe reword to just document that it's testing partial date mixin via borrow, since it needs a concrete case?

Reply:
Revising accordingly.